objectPerformance/src/quality_obj.py

```diff
@@ -63,6 +63,20 @@ def __init__(self, ak_arrays, obj: str):
                              & (ak_arrays['region'] == 1))
         self.sel_not_4 = (ak_arrays['region'] == 4)
 
+        ## EG IDs from 125x
+        # for EG: region == HGC
+        self.sel_EG_barrelID = (ak_arrays['region'] == 0) & (ak_arrays['passeseleid'] == 1)
+        self.sel_EG_endcapID = (ak_arrays['region'] == 1) & (ak_arrays['passessaid'] == 1)
+
+        # for EG: quality = HwQual, alt approach: use HW qual bits directly instead of the menu ntuple variables
+        #self.sel_EG_barrelID = (ak_arrays['region'] == 0) & (((ak_arrays['quality'] >> 1)&1) == 1)
+        #self.sel_EG_endcapID = (ak_arrays['region'] == 1) & (((ak_arrays['quality'] >> 0)&1) == 1)
+
+    @property
+    def QUAL_125x_EGID(self):
+        return ~(self.sel_EG_barrelID | self.sel_EG_endcapID)
```

Comment:
@artlbv as we discussed on MM, `turnon_collection.py` takes the `QUAL_*` objects with a negation. So my question is whether this negation here is wanted (it would result in a double negation in `turnon_collection.py`.

Reply:
yes, the `sel_EG_barrelID` are `True` if the object passes the ID -> we want to use these.

Reply:
Ok, understood! Maybe we want to change the definition of all the quality objects in `Quality` for consistency, so that we can also drop the negation in the application of this selection in `turnon_collection.py`

Reply:
Ok, let's do that in another PR then :)